src/cursor/cur_layered.c

```diff
@@ -2057,53 +2028,158 @@ __clayered_next_random(WT_CURSOR *cursor)
     API_END_RET(session, ret);
 }
 
+/*
+ * __clayered_modify_leader --
+ *     Apply a modify on a leader node.
```

Comment:
np: would it make sense to make the description more specific?

```suggestion
 *     Apply a modify on a leader node. Effectively calls modify for the stable constituent.
```

Reply:
Good call! Thanks, done in `86e96509c9`